### ISSUE TITLE
Cauldron on GitHub menu item is not visible properly when page is Re flowed

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#566]
size your browser down to ~320px width...

![image](https://user-images.githubusercontent.com/3180185/159587030-3e79ea5c-f100-4f46-aa0c-a020cb55f017.png)

<dl><dt>Description</dt><dd>Content disappearing and not being available when page has re-flowed at 320px. 

Element : Cauldron on GitHub Menuitem on header is not visible when page is Re flowed.</dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
Content MUST be presented without loss of information or functionality when the viewport is set to 320 CSS pixels wide.

HOW TO FIX: 
Fix this issue by ensuring content does not overlap other content when the screen width is set to 320px by using techniques such as CSS media queries, grids, or flexbox to reflow content down to 320px width.

REFERENCE: 
Deque University:https://dequeuniversity.com/class/responsive-zoom2/zoom/linear-text-reflow
https://dequeuniversity.com/class/responsive-zoom2/responsive/
WCAG Understanding document:
https://www.w3.org/WAI/WCAG21/Understanding/reflow

BACKGROUND: 
People with low vision often read content on a desktop computer by scaling content using the browser zoom - up to 400%. When zooming causes content to overlap other content, users with low vision may not be able to see or use that content. When content is created so that it reflows without losing information or functionality at screen widths down to 320px, users can view a web page in a 1280px width browser screen and zoom the content to 400% (320px) without having to scroll side to side to read and without losing access to content.</dd><dt>Source code</dt><dd>

```html
<li class="MenuItem--separator" tabindex="0" role="menuitem"><a href="https://github.com/dequelabs/cauldron" class="fa fa-github" aria-label="Cauldron on GitHub" tabindex="-1"></a></li>
```
</dd><dt>Checkpoint</dt><dd>Reflow (1.4.10.a)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/</dd></dl>

**Comment by YuriDevAT** [comment]
Hi. I would like to work on this issue.
